app/models/spree/ability_decorator.rb

```diff
@@ -155,6 +155,8 @@ def add_product_management_abilities(user)
     can [:admin, :index, :read, :search], Spree::Taxon
     can [:admin, :index, :read, :create, :edit], Spree::Classification
 
+    can [:admin, :index, :import, :save], ProductImporter
```

Comment:
That's including E2E permissions and updating items for multiple enterprises from one spreadsheet. If they don't have the permissions it doesn't update that product (and explains why in the pre-save feedback, with line number etc).